Assets/PatchKit Patcher/Scripts/Debug/DebugMenu.cs

```diff
@@ -0,0 +1,216 @@
+﻿using System;
+using System.Diagnostics;
+using PatchKit.Unity.Patcher;
+using PatchKit.Unity.Patcher.Debug;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace PatchKit_Patcher.Scripts.Debug
+{
+    public class DebugMenu : MonoBehaviour
+    {
+        private static readonly DebugLogger DebugLogger = new DebugLogger(typeof(DebugMenu));
+
+        private Rect _popUp, _debugMenu;
```

Comment:
Please rename to `_popupRect` & `_rect` to make the names precise.

Reply:
done